system/web/services/SchedulerService.cfc

```diff
@@ -195,4 +195,19 @@ component extends="coldbox.system.web.services.BaseService" accessors="true" {
 		return false;
 	}
 
+	/**
+	 * Restarts a scheduler from this manager, if it exists.
+	 *
+	 * @name The name of the scheduler
+	 *
+	 * @return True if restarted, false if not found
+	 */
+	boolean function restartScheduler( required name ){
```

Comment:
Can you make sure you add the force and timeout arguments here so they can flow down to the scheduler itself.  Thanks